projects/openapi-io/src/validation/validation-schemas.ts

```diff
@@ -1,5 +1,6 @@
 const openapi3_0_schema_object = {
   type: 'object',
+  'x-custom-validator': 'validateSchema',
```

Comment:
whenever we add a new custom validator, we can anchor them using an extension in their validation schemas